lib/rake/application.rb

```diff
@@ -604,5 +604,9 @@ def rakefile_location backtrace = caller
 
       backtrace.find { |str| str =~ re } || ''
     end
+
+  private
+    FIXNUM_MAX = (2**(0.size * 8 - 2) - 1) # :nodoc:
+    
```

Comment:
Hey @michaeljbishop - the `private` declaration doesn't actually have the intended effect of making `FIXNUM_MAX` a private constant; see discussion in ruby/rake#80 for more context/investigation, in case you want to chip in, or vote up / vote down my proposed "fix" ... :smiley: